lib/modules/datasource/packagist/schema.ts

```diff
@@ -17,6 +54,12 @@ export const ComposerRelease = z
           .nullable()
           .catch(null),
         time: z.string().nullable().catch(null),
+        require: z
+          .object({
+            php: z.string(),
+          })
+          .nullable()
+          .catch(null),
```

Comment:
```suggestion
```

I suggest we separate out the `require.php` gathering into a future PR

Reply:
Although if you think that's too painful, I don't mind keeping it

Reply:
Haha, no, I just missed it